backend/tests/integration-async/disambiguation/test_disambiguation_tasks.py

```diff
@@ -630,3 +630,63 @@ def assert_disambiguation_task():
         )
 
     retry_until_pass(assert_disambiguation_task)
+
+
+def test_disambiguate_authors_create_two_author_with_same_name(
```

Comment:
I actually meant doing this through the disambiguation, to have a test of everything that's going on. So create a paper with two authors with same names (with no matching author record), and check that works correctly.

Reply:
So I did it, in test I created a record with two authors with same names for which new authors are created (what is tested in assertion)